README.adoc

```diff
@@ -7,16 +7,18 @@ https://ci.appveyor.com/project/seanieyap/main-r8jp5[image:https://ci.appveyor.c
 https://app.netlify.com/sites/planmysem/deploys[image:https://api.netlify.com/api/v1/badges/6c19b45b-dbdd-4eff-afe2-200356d6ebb4/deploy-status[Build Status]]
 https://coveralls.io/github/CS2113-AY1819S2-T08-3/main?branch=master[image:https://coveralls.io/repos/github/CS2113-AY1819S2-T08-3/main/badge.svg?branch=master[Coverage Status]]
 https://www.codacy.com/app/CS2113T-T08-3/main?utm_source=github.com&utm_medium=referral&utm_content=CS2113-AY1819S2-T08-3/main&utm_campaign=Badge_Grade[image:https://api.codacy.com/project/badge/Grade/90f57d16da5049858ef67b6eaaf692f2[Codacy Badge]]
+https://opensource.org/licenses/MIT[image:https://img.shields.io/badge/License-MIT-yellow.svg[License: MIT]]
 
 image::Ui.png[]
 
-PlanMySem is a CLI (Command Line Interface) scheduling/calendar application that targets NUS students and staff who prefer to use a desktop application for managing their schedule/calendar.
-PlanMySem automatically creates a planner that is synchronised according to the NUS academic calendar for the current semester and enables easy creation, editing and deleting of items.
-Special weeks such as recess week and reading week are taken into account within our unique recursion system.
-Items can then be efficiently managed via the intuitive tagging system.
+Welcome to PlanMySem! Choose a topic from the table of contents to learn about available features with examples, find answers or view a summary of the commands.
```

Comment:
This part should remain in User Guide because got no table of contents in Readme